### ISSUE TITLE
Issues with PipeListener

### ISSUE THREAD

**Issue by Deixel** [Deixel/DeixUtils#2]
Starting a PipeListener without anything waiting in the pipe causes a hang/crash.
Workaround on server start is to echo something to the pipe before starting the server, but would be nice to work out what's causing it so discord integration can be properly started at will.

**Comment by Deixel** [comment]
This seems to be caused by something hanging in FileInputStream
```
java.base@11.0.9.1/java.io.FileInputStream.open0(Native Method)
java.base@11.0.9.1/java.io.FileInputStream.open(FileInputStream.java:219)
java.base@11.0.9.1/java.io.FileInputStream.<init>(FileInputStream.java:157)
java.base@11.0.9.1/java.io.FileInputStream.<init>(FileInputStream.java:112)
uk.co.deixel.deixutils.PipeListener.<init>(PipeListener.java:21)
uk.co.deixel.deixutils.DeixUtils.startDiscord(DeixUtils.java:87)
uk.co.deixel.deixutils.commands.ToggleDiscord.onCommand(ToggleDiscord.java:20)
```

**Comment by Deixel** [comment]
Fixed in 16cdfa7fbf43b965fdd08a3be5d711848a6b9502
